src/app/app.service.ts

```diff
@@ -16,6 +16,7 @@ export class AppService {
     /**
      * Sets up app in a way that can be used by main.ts and e2e tests
      */
+    // eslint-disable-next-line @typescript-eslint/require-await
     public static async setup(app: INestApplication): Promise<void> {
```

Comment:
This is a good question. No idea why this was made `async`, but I'm hesitant to change it to a synchronous function.

Reply:
I'm fairly certain it was made `async` because this code was copied from somewhere else where it was important that it be async. You should feel comfortable changing it in this case.